src/server/page_server.cpp

```diff
@@ -421,7 +547,91 @@ int
 page_server::followee_connection_handler::send_receive (follower_to_followee_request reqid, std::string &&payload_in,
     std::string &payload_out)
 {
-  return m_conn->send_recv (reqid, std::move (payload_in), payload_out);
+  const css_error_code error_code =  m_conn->send_recv (reqid, std::move (payload_in), payload_out);
+  if (error_code != NO_ERRORS)
+    {
+      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_CONN_PAGE_SERVER_CANNOT_BE_REACHED, 0);
+      return ER_CONN_PAGE_SERVER_CANNOT_BE_REACHED;
+    }
+
+  return NO_ERROR;
+}
+
+int
+page_server::followee_connection_handler::request_log_pages (LOG_PAGEID start_pageid, int count,
+    std::vector<LOG_PAGE *> &log_pages_out)
```

Comment:
`const std::vector<LOG_PAGE *> &log_pages_out`

Reply:
Thank you. I'll fix them in the following PR.